src/status_im/contexts/profile/contact/header/view.cljs

```diff
@@ -14,15 +14,14 @@
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
-(defn on-contact-request
+(defn on-send-contact-request
   []
-  (rf/dispatch [:show-bottom-sheet
-                {:content (fn [] [contact-request/view])}]))
+  (let [main-event [:show-bottom-sheet {:content contact-request/view}]]
+    (rf/dispatch [:profile/check-profile-update-prompt main-event])))
```

Comment:
Shows the sheet to update the profile when the user sends a contact request :eyes: